cmdutils/switches/switches_test.go

```diff
@@ -16,26 +16,36 @@ package switches
 import (
```

Comment:
Will we also have a test for the new behavior?

Reply:
It's already there, adapted old test and renamed it to `should disable all controllers when no flag is passed`